rules/o365/exfiltration_o365_exchange_transport_rule_creation.toml

```diff
@@ -0,0 +1,49 @@
+[metadata]
+creation_date = "2020/11/18"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/11/18"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies a transport rule creation in Office 365. Exchange Online mail transport rules should be set to not forward
+email to domains outside of your organization. An adversary may create transport rules to exfiltrate data.
```

Comment:
Is it fair to say that some organizations might have a valid use case for doing this, but it may go against best practices?

Reply:
I think that sounds accurate. I'll add it to mention its a best practice.  👍